apps/retrospective/services/sprint_sync.go

```diff
@@ -303,19 +304,22 @@ func (service SprintService) AssignPoints(sprintID string) (err error) {
             (tasks.estimate - (SUM(sprint_member_tasks.points_earned) OVER
 				(PARTITION BY sprint_tasks.task_id)) + (SUM(sprint_member_tasks.points_earned) OVER
 				(PARTITION BY sprint_tasks.id))) AS remaining_points
-        `).
-		QueryExpr()
+        `).QueryExpr()
 
-	err = db.Exec(`
-    UPDATE sprint_member_tasks
+	updateSQL := `UPDATE sprint_member_tasks
 	    SET points_assigned = COALESCE(sprint_member_tasks.time_spent_minutes
 				/ NULLIF(s1.sprint_task_total_time_spent, 0) * s1.remaining_points, 0),
             points_earned = COALESCE(sprint_member_tasks.time_spent_minutes
 				/ NULLIF(s1.sprint_task_total_time_spent, 0) * s1.remaining_points, 0),
             updated_at = NOW()
         FROM (?) AS s1 
-        WHERE s1.sprint_id = ? and sprint_member_tasks.id = s1.id
-    `, dbs, sprintID).Error
+        WHERE s1.sprint_id = ? and sprint_member_tasks.id = s1.id`
+
+	if sprintTaskID != nil {
+		updateSQL = fmt.Sprintf("%v AND s1.sprint_task_id = %v", updateSQL, *sprintTaskID)
```

Comment:
why are we using `%v`? Is `%s` usable?
Also don't append `sprintTaskID` here, that may lead to query injection. Append that in line 322, I think gorm should ignore extra params, if not you need to have two separate calls